bms/bms_app/views.py

```diff
@@ -81,7 +82,11 @@ def create_product(request):
     else:
         data = request.POST
         data1 = request.FILES
-
+        if request.FILES:
+            data1['prod_file'] = request.FILES['prod_file']
```

Comment:
Varibale name should be self explanatory, change varibale name "data1" to file_data or something else